.github/workflows/swagger-json.yml

```
@@ -0,0 +1,85 @@
+      - name: Copy Swagger JSON
+        run: |
+          cp common-api.json amrit-docs/docs/swagger/common-api.json
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ secrets.DOCS_REPO_TOKEN }}
+          path: amrit-docs
+          branch: auto/swagger-update
+          base: main
+          base-repository: PSMRI/AMRIT-Docs
```

actionlint (1.7.10)

[error] input "base-repository" is not defined in action "peter-evans/create-pull-request@v6". available inputs are "add-paths", "assignees", "author", "base", "body", "body-path", "branch", "branch-suffix", "commit-message", "committer", "delete-branch", "draft", "git-token", "labels", "milestone", "path", "push-to-fork", "reviewers", "signoff", "team-reviewers", "title", "token"

(action)